Frameworks/UIKit/StarboardXaml/StarboardXaml.h

```diff
@@ -30,6 +38,8 @@ typedef enum {
     ActivationTypeFile = 5,
 } ActivationType;
 
+IInspectable* GetRootXamlElement();
```

Comment:
Can we move this over to XamlCompositor.h/.cpp, and into the XamlCompositor namespace?